azure-keyvault/src/main/java/com/microsoft/azure/keyvault/models/DeletionRecoveryLevel.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Code generated by Microsoft (R) AutoRest Code Generator 1.2.2.0
+ * Changes may cause incorrect behavior and will be lost if the code is
+ * regenerated.
+ */
+
+package com.microsoft.azure.keyvault.models;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+/**
+ * Defines values for DeletionRecoveryLevel.
+ */
+public final class DeletionRecoveryLevel {
```

Comment:
This is a model from the updated spec we should remove it from the refactoring, and add back in a subsequent update.